test/model/store_test.dart

```diff
@@ -1082,6 +1082,22 @@ void main() {
         checkLastRequestFcm(token: '456def');
       }
     }));
+
+
+    test('fallback to default appBundleId in case packageInfo is null', ()async{
+      // This tests the case where packageInfo is null
+      // and default appBundleId gets passed to the [NotificationService.registerToken] method
```

Comment:
Ideally the comment shouldn't be needed here, as the test code itself would be self-explanatory in most cases.

It would be better to have comments on the exact places where something might not be obvious to a reader.